doc/usecases/plugins/spec_globbing.md

```diff
@@ -0,0 +1,47 @@
+# Use Case: Specification for REST application communicating with multiple services
+
+## Summary
+
+- **Title:** Specification for REST application communicating with multiple services
+- **Scope:** Configuration
+- **Level:** Developer Goal
+- **Actors:** User (usually a application developer)
+- **Brief:** User configures specifcation for configuration of REST application.
+
+## Scenarios
+
+- **Precondition:** kdb is installed.
+- **Main success scenario:** Specification is written and stored system wide or user specifc. All services which are added in the configuration need to adhere to this configuration specification.
+- **Alternative scenario:** User is limited to only using one service if globbing is not used. 
+- **Error scenario:** Technical problems while writing the specification.
+  The user is informed about the problem.
```

Comment:
Write which globbings are not allowed (what means technical problem). What is with overlaps, e.g. that a specific service has a different configuration?